templates/core/terraform/appgateway/appgateway.tf

```diff
@@ -0,0 +1,61 @@
+resource "azurerm_public_ip" "appgwpip" {
+  name                  = "pip-agw-${var.resource_name_prefix}-${var.environment}-${var.tre_id}"
+  resource_group_name   = var.resource_group_name
+  location              = var.location
+  allocation_method     = "Static"
+  sku                   = "Standard"
+}
+
+resource "azurerm_application_gateway" "agw" {
+  name                = "agw-${var.resource_name_prefix}-${var.environment}-${var.tre_id}"
+  resource_group_name = var.resource_group_name
+  location            = var.location
+
+  sku {
+    name     = "Standard_v2"
+    tier     = "Standard_v2"
+    capacity = 2
```

Comment:
Do we need 2? Is 1 not sufficient?

Reply:
Yes, Standard_v2 SKU is required by Standard public IP. (Old) Standard SKU only allows for a Basic public IP.

Reply:
Not the SKU, capacity unit.

Reply:
Maybe make a variable with a default of 1?